critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -90,6 +90,14 @@ def test_create(self):
         self.assert200(response)
         self.assertIn(self.review_text, str(response.data))
 
+    def test_create_duplicate(self):
+        review = self.create_dummy_review()
+
+        self.temporary_login(self.user)
+        response = self.client.get('/review/write?release_group=%s' % review["entity_id"],
```

Comment:
Small nitpick, Using `'` and `"` on the same line. Pick one. :)

Reply:
Solved, and squashed all the changes into one commit.